assets/js-translations/teacher.fi.json

```diff
@@ -29,6 +29,30 @@
   "Student name":
   "Opiskelijan nimi",
 
+  "First Name":
+  "Etunimi",
+
+  "Last Name":
```

Comment:
Does Sisu require the text to have both words capitalized? Usually in A+, the Finnish protocol of capitalizing only the first word has been used.

Reply:
Right. Fixed.